frontend/src/lib/components/LemonButton/LemonButton.scss

```diff
@@ -1,3 +1,4 @@
+@use "sass:color";
```

Comment:
Not a fan of this. Every extra thing we import 1) slows down builds, 2) complicates builds by requiring even more custom plugins, 3) makes it harder to migrate to any other CSS framework down the line. 

Let's defer as much as we can to CSS, instead of the build scripts. Hence, could you either use the CSS brightness filter, or then hardcode the `--primary-light` and `--primary-lighter` vars so that we can reuse them instead of hoping that the `-5` and `-10` lightness pattern will be respected everywhere else.